### PR TITLE
Blending Orcs

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- Added Green, Brown, Gray and Pale Orcish ethnicities.

# How to test:
Check them in-game, randomize in the Ruler Designer and tell your opinion.

### CODE REVIEW

**Review by TheGroxEmpire — COMMENTED** [review]
- I think their ears need to have pointier top tip:

<details>
<summary>In-game model</summary>

image
</details>
<details>
<summary>Reference</summary>

image
image

</details>

- It would be nice to have the option for shorter tusks like in the reference image above.

- Some clipping issues on ears for female Orcs with these two hair style.
<details>

image
image

</details>

- I don't see any bald Orcs beside the Pale Orcs on in-game or ruler designer. Maybe add balding to Orcs like how you did it with Dwarves.

File: common/culture/cultures/orc.txt [hunk 1]
```diff
@@ -496,9 +464,6 @@
 	}
 
 	laughting_skull = {
```
Comment (TheGroxEmpire) [inline common/culture/cultures/orc.txt:466]: Minor Typo:
```suggestion
	laughing_skull = {
```

File: common/culture/cultures/orc.txt [hunk 2]
```diff
@@ -923,9 +859,6 @@
 	}
 
 	moknathal = {
```
Comment (TheGroxEmpire) [inline common/culture/cultures/orc.txt:861]: I think Mok'nathal should have their own model. They are described as taller and bigger than Orcs (https://wow.gamepedia.com/Mok%27nathal#Appearance)

If you look at Rexxar new model without the mask on, he has very distinct Ogre features that none of the Orcs have:
<details>

image
</details

Comment (zumbak04) [reply common/culture/cultures/orc.txt:861]: I plan them later once the ogres are done.

File: common/culture/cultures/orc.txt (hunk 1 above)
Comment (zumbak04) [reply common/culture/cultures/orc.txt:466]: This feeling when fixing a typo may break more things in the future. I wouldn't fix this.

**Comment by ulricee** [comment]
In warcraft, male and female orcs suffer an enormous sexual dysmorphia. I have no idea if it's addable but seeing it in game would be interesting.

Other than that I see no problem

**Comment by zumbak04** [comment]
> In warcraft, male and female orcs suffer an enormous sexual dysmorphia. I have no idea if it's addable but seeing it in game would be interesting.

Hello! Explain what do you think should be changed and a reference image plz.

**Comment by ulricee** [comment]
Mostly facial structure.
Here's what's is showed in WoW (male and female exemple):
https://cdna.artstation.com/p/assets/images/images/000/279/588/large/orcmale.jpg?1414798722
https://im.ziffdavisinternational.com/ign_ap/screenshot/u/updated-wow-orc-female-model-revealed/updated-wow-orc-female-model-revealed_qvs7.jpg

In CK3 the woman share the same Jawline and jagged look as the men which isn't the case in WoW (also their's the hunchback but that's another thing) where the woman possess somehow way softer features (male and female exemple):




I have absolutely no idea how to "fix" this but It's maybe something you should look into.

**Comment by zumbak04** [comment]
> * Some clipping issues on ears for female Orcs with these two hair style.

I fixed the first one, but I don't get what is wrong on the second one.
The ear goes through the hairs.
<details>
<summary>Click to expand</summary>

image
image

</details>

**Comment by zumbak04** [comment]
> * I think their ears need to have pointier top tip:

Changed in bb37c4f693cfe72a44a0944e7c9866372d1a5b58.

> * It would be nice to have the option for shorter tusks like in the reference image above.

Added in 9141ef6ce583df96b74dd35acdc460d08e778edf.

> * I don't see any bald Orcs beside the Pale Orcs on in-game or ruler designer. Maybe add balding to Orcs like how you did it with Dwarves.

Added in efa8ff59d93b9308a19b1d2c13361fa4320f86b2.

**Comment by zumbak04** [comment]
> In CK3 the woman share the same Jawline and jagged look as the men which isn't the case in WoW or at least in a way more limited fashion (also their's the hunchback but that's another thing) where the woman possess somehow way softer features (male and female example):

I pointed the word jaw, so I limited female jaw in b9ab3f63f30ef274ae472774a671e0ff731d0d77.

**Comment by ulricee** [comment]
the problem seems fixed thank you

**Comment by ulricee** [comment]
Found another problem, The Beauty trait make the teeth of orc male clip through their mouth on random characters.

**Review by Deuce20201 — APPROVED** [review]
I am not seeing the clipping issues ulricee has pointed out. 

Edit: Well Isee it in his pictures. But when I randomized appearances for males/females on my end I saw nothing of it.

**Comment by zumbak04** [comment]
> Found another problem, The Beauty trait make the teeth of orc male clip through their mouth on random characters.

Fixed in 09ccd62707f8e6b2b0b23ae703ab195f72ea81ae.

**Comment by ulricee** [comment]
Seems fixed

**Review by Deuce20201 — APPROVED** [review]
Forget my comment before. Apparently I'm a moron and couldn't read ulricee's comment properly :P (about the beauty trait causing the clipping).

I noticed zumbak fixed it. There is no clipping issue now. With the beauty trait.

**Review by ulricee — APPROVED** [review]

**Review by TheGroxEmpire — COMMENTED** [review]
The new features you added are working properly: I could see bald Orcs in-game and small tusk option. The new ear tips look good also.

There's a couple minor issues on the hairstyle that you fixed. This doesn't happen often and only on this hairstyle alone:
<details>

image

</details>

**Comment by zumbak04** [comment]
> There's a couple minor issues on the hairstyle that you fixed. This doesn't happen often and only on this hairstyle alone:

Vanilla clipping, I reported it to Paradox.
https://forum.paradoxplaza.com/forum/threads/clipping-issues-when-combining-outfits-or-headwear-share-them-here.1416571/post-27212210

**Review by TheGroxEmpire — APPROVED** [review]
All good to me then!

**Pull request merged**